### ISSUE TITLE
Add shortcuts

### ISSUE THREAD

**Issue by StaffEngineer** [StaffEngineer/velo#157]
Add shortcuts for common actions (new note, delete note, etc).a

**Comment by irfanabliz** [comment]
Hi, sorry I don't know where should I write this so Im gonna write it here.

I'm trying to create a keyboard short cut for deleting a selected node, I am kind of lost 😅

this is what I'm trying to do it, and I'm stuck here like how can I access that `velo_node_query` or is it even possible, reading the doc on bevy didn't really help or I maybe I wasn't looking at the right place

![image](https://github.com/StaffEngineer/velo/assets/17282448/a9678a6b-a410-4008-962e-f2c918e2bd4c)

is this even the right approach? 

thanks

**Comment by irfanabliz** [comment]
another question, also what short cuts is good for adding Circle and Rectangle? cmd + c its for copy, cmd + r for Rectangle?

**Comment by StaffEngineer** [comment]
@irfanabliz 

Yeah, you have to learn bevy to contribute to velo. There are systems - methods that run in specific order (schedule). Systems work on entities, entities can have any number of components. You can add any query to system. So yeah, on some key press you just check that this entity is selected and despawn it, so your approach looks fine to me.

cmd + c is for copy text/node, cmd + r for rectangle perhaps is fine. I am not sure about which shortcuts are the best, make sure that they don't interfere with basic functionality like copy/pasting, etc.
